### DISCUSSION TITLE
Brick wall

### DISCUSSION THREAD

**Discussion by mstraus66** [blakeblackshear/frigate#2930] [Ask A Question]
Hi all,

I have been struggling with Frigate for quite some time now and I was wondering if anybody there would like to take me under their wings and get it up and running. I cannot get my frigate.yml going without some errors I just dont understand.

This ia very much a work in progress so please dont judge me too hard ;-)

I am running hass on a tinkerboard.

mqtt:
  host: 192.168.1.21

cameras:
  deurbel:
    ffmpeg:
      inputs:
        - path: rtsp://marc:Brunhilde01@192.168.1.44:88/videoMain
          roles:
            - stream
#            - detect
#            - rtmp
#  detect:
#    height: 1440
#    width: 2560
#rtmp:
#  enabled: False

  voordeur:
#    ffmpeg:
      inputs:
        - path: rtsp://marc:Brunhilde01@192.168.1.43:88/videoMain
          roles:
            - stream
#            - detect
 #           - rtmp
 # detect:
 #   height: 1080
 #   width: 1920
#rtmp:
#  enabled: False
#
  voortuin:
#    ffmpeg:
      inputs:
        - path: rtsp://marc:Brunhilde01@192.168.1.42:88/videoMain
          roles:
            - stream
#           - detect
#           - rtmp
#  detect:
#    height: 960
#    width: 1280
#rtmp:
#  enabled: False

Any help or hints will be greatly appreciated. Of course I have scoured Google first. Extensively!

This is the output from the log:
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[2022-03-09 09:58:52] frigate.app                    INFO    : Starting Frigate (0.10.0-bfecee9)
*************************************************************
*************************************************************
***    Your config file is not valid!                     ***
***    Please check the docs at                           ***
***    https://docs.frigate.video/configuration/index     ***
*************************************************************
*************************************************************
***    Config Validation Errors                           ***
*************************************************************
'ffmpeg'
Traceback (most recent call last):
  File "/opt/frigate/frigate/app.py", line 312, in start
    self.init_config()
  File "/opt/frigate/frigate/app.py", line 77, in init_config
    user_config = FrigateConfig.parse_file(config_file)
  File "/opt/frigate/frigate/config.py", line 903, in parse_file
    return cls.parse_obj(config)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/main.py", line 511, in parse_obj
    return cls(**obj)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/main.py", line 329, in __init__
    values, fields_set, validation_error = validate_model(__pydantic_self__.__class__, data)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/main.py", line 1022, in validate_model
    v_, errors_ = field.validate(value, values, loc=field.alias, cls=cls_)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/fields.py", line 856, in validate
    v, errors = self._validate_mapping_like(v, values, loc, cls)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/fields.py", line 991, in _validate_mapping_like
    value_result, value_errors = self._validate_singleton(v_, values, v_loc, cls)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/fields.py", line 1064, in _validate_singleton
    value, error = field.validate(v, values, loc=loc, cls=cls)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/fields.py", line 854, in validate
    v, errors = self._validate_singleton(v, values, loc, cls)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/fields.py", line 1071, in _validate_singleton
    return self._apply_validators(v, values, loc, cls, self.validators)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/fields.py", line 1118, in _apply_validators
    v = validator(cls, v, values, self, self.model_config)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/class_validators.py", line 313, in <lambda>
    return lambda cls, v, values, field, config: validator(v)
  File "/usr/local/lib/python3.8/dist-packages/pydantic/main.py", line 676, in validate
    return cls(**value)
  File "/opt/frigate/frigate/config.py", line 547, in __init__
    if len(config["ffmpeg"]["inputs"]) == 1:
KeyError: 'ffmpeg'
*************************************************************
***    End Config Validation Errors                       ***
*************************************************************
[cmd] python3 exited 1
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.

**Comment by blakeblackshear** [comment]
Please open an issue rather than a discussion. The template requests the info needed to provide support. https://github.com/blakeblackshear/frigate/issues/new/choose

**Comment by mstraus66** [comment]
Thanks for the increadible fast reaction!!

 

 

Met vriendelijke groet,

 

 

Marc R. Straus MBA 

Karmozijnhof 7 

1339 HN  Almere Buiten 

The Netherlands 

+31 615295893

***@***.*** ***@***.***> 

 

 

 

Van: Blake Blackshear ***@***.***> 
Verzonden: woensdag 9 maart 2022 12:50
Aan: blakeblackshear/frigate ***@***.***>
CC: mstraus66 ***@***.***>; Author ***@***.***>
Onderwerp: Re: [blakeblackshear/frigate] Brick wall (Discussion #2930)

 

Please open an issue rather than a discussion. The template requests the info needed to provide support. https://github.com/blakeblackshear/frigate/issues/new/choose

—
Reply to this email directly, view it on GitHub <https://github.com/blakeblackshear/frigate/discussions/2930#discussioncomment-2323525> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AUFGNC2GK7KSVLAYZ4UJFWDU7CF7XANCNFSM5QI36NLQ> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .
